src/openzaak/components/catalogi/management/commands/import.py

```diff
@@ -39,6 +40,8 @@ def add_arguments(self, parser):
 
     @transaction.atomic
     def handle(self, *args, **options):
+        backend = requests_cache.BaseCache()
+        requests_cache.install_cache("import", backend=backend)
```

Comment:
Does this cache to filesystem or...? I'm going to need some more information :P

I'd also rather see this as a decorator or context manager that guarantees the cache is uninstalled again even if something unexpectedly throws exceptions. it seems this is globally installing the cache by monkeypatching, which lives across multiple requests in the uwsgi process. That also raises questions for me whether this is thread safe or not?

I think I'd also like to see this tied in with Django's cache framework so we can point to a cache backend from the django settings (which opens up the option to store stuff in memory / redis etc. down the road).

Reply:
I think the default is in-memory (https://requests-cache.readthedocs.io/en/stable/modules/requests_cache.backends.base.html), I'll look into tieing this into the existing Django cache framework

Reply:
yes, redis is (by design) not thread-safe. We'd opt for a local memory cache in this case, or if a request object is available, on the request instance even (I believe we already have such a dirty hack)

Reply:
I switched to LocMemCache